lib/handlers_v5.js

```diff
@@ -341,3 +344,21 @@ function shutdown_request(request) {
 
     status_idle.call(this, request);
 }
+
+function is_complete_request(request) {
```

Comment:
Yes, languages with semantic indentation (like CoffeeScript) wouldn't be indented properly.

Let's play safe in this PR and respond `unknown` if `this.session.transpile` is defined, i.e.:

```js
if (this.session.transpile) {
    request.respond(this.shellSocket, "is_complete_reply", {status: "unknown"});
    return;
}

// use esprima
```